### DISCUSSION TITLE
Does react-navigation v5 supports React Native 72 ?

### DISCUSSION THREAD

**Discussion by FulfilledMobileDev** [react-navigation/react-navigation#11764] [General]
Currently our mobile app version is 68 and we are trying to update our app to React Native 72. @rnx-kit/align-deps tools recommends to update react-navigation from v5 to v6. 
But from the docs I do not see such requirement and assume v5 should work with React native 72 version. 

We do have plans to move to v6 in short-term but trying to get this query clarified from community.

**Comment by mthomas100** [comment]
Also have the same question
